docs/release-notes/RELEASE-NOTES-4.2.1.md

```diff
@@ -4,6 +4,7 @@
 It includes the following changes:
 * Change to a generic error message for the inner transaction of a GAMetaTX.
 * Long overdue bump of info-field in key block header from pre-fortuna value.
+* Improves State Channels tests to cover `Call.origin` and `Call.caller`
```

Comment:
Shouldn't this line summarize the changes instead of referencing tests?

Reply:
I agree